### ISSUE TITLE
Issue during crew install

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#6550]
```
Updating crew package information...

sh: 1: perl: not found
Installing core Chromebrew packages...
```

**Comment by satmandu** [comment]
During a fresh install?

**Comment by uberhacker** [comment]
Yes, apparently something is trying to use perl before that package is installed.

**Comment by satmandu** [comment]
Grep core packages for perl?

**Comment by uberhacker** [comment]
How about install perl before attempting to use it? ;)

**Comment by satmandu** [comment]
The problem here is that `crew update compatible` in `install.sh` is running all non-`self` section commands in package files. 
`info2man.rb` has ```  @_perl_version = %x[echo "printf '%vd', $^V;" | perl].to_s```
@saltedcoffii FYI...

**Comment by uberhacker** [comment]
Well, we shouldn't be running `crew update` in install.sh before all the core packages are installed anyways.  Time to swap some lines around in install.sh?

**Comment by uberhacker** [comment]
I see you created a PR to fix this in #6552.
